### PR TITLE
DAOS-9355 doc: release notes for 2.2.0

### PR DESCRIPTION
**Pull request by Michael-Hennecke**
draft release notes for 2.2.0

Doc-only: true
Signed-off-by: Michael Hennecke <michael.hennecke@intel.com>

### CODE REVIEW

File: docs/release/release_notes_v2_2.md [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+- The required number of hugepages on DAOS servers is now auto-calculated.
+  It is no longer necessary (and discouraged) to specify `nr_hugepages` in the
+  `daos_server.yml` configuration file.
+
+- Intel VMD devices are now supported in the control plane.
+  Roadmap items exist for future DAOS releases to add
+  support for LED management of VMD-managed NVMe disks,
+  and hot-plug support for VMD-managed NVMe disks.
+
+- The `dmg pool create` command now accepts pool size specification in percent.
```
Comment (ipoddubn) [inline docs/release/release_notes_v2_2.md:55]: suggest that we state that it's "in addition to using the arbitrary numbers"

File: docs/release/release_notes_v2_2.md [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+- POSIX containers (DFS) now support file modification time (mtime).
+
+### Known Issues and limitations
+
+- [DAOS-11317](https://daosio.atlassian.net/browse/DAOS-11317)
+  Running the Mellanox-provided `mlnxofedinstall` script to install a new version of MLNX\_OFED,
+  while the `mercury-ucx` RPM is already installed, will un-install `mercury-ucx`
+  (as well as mercury-ucx-debuginfo if the debuginfo RPMs are installed).
+  This leaves DAOS non-functional after the MOFED update.
+  The workaround is to run `{yum|dnf|zypper} install mercury-ucx [mercury-ucx-debuginfo]`
```
Comment (ipoddubn) [inline docs/release/release_notes_v2_2.md:66]: I suggest using a more formal approach, i.e.

Issue: blah-blah
Workaround: blah

For the known problem without a workaround, a state "No workaround is available at this point" (or "no known workaround is available")

File: docs/release/release_notes_v2_2.md [hunk 3]
```diff
@@ -0,0 +1,128 @@
...
+- Add VPIC and LAMMPS applications to soak testing framework.
+- Report pool global version on dmg pool list/query.
+- Reject pool connection from old clients after pool upgrade.
+
+## Additional resources
+
+Visit the [online documentation](https://docs.daos.io/v2.2/) for more
+information. All DAOS project source code is maintained in the
+[https://github.com/daos-stack/daos](https://github.com/daos-stack/daos) repository.
+Please visit this [link](https://github.com/daos-stack/daos/blob/master/LICENSE)
```
Comment (ipoddubn) [inline docs/release/release_notes_v2_2.md:123]: should the link be pointing to the 2.2 branch instead of the master?

Comment (Michael-Hennecke) [reply docs/release/release_notes_v2_2.md:123]: done

File: docs/release/release_notes_v2_2.md (hunk 1 above)
Comment (Michael-Hennecke) [reply docs/release/release_notes_v2_2.md:55]: done

File: docs/release/release_notes_v2_2.md [hunk 4]
```diff
@@ -0,0 +1,134 @@
...
+## DAOS Version 2.2.0 (2022-09-xx)
+
+### General Support
+
+This release adds the following changes to the DAOS support matrix:
+
+- Rocky Linux 8 support is added.
+- CentOS Linux 8 support is removed.
+- Support for the `libfabric/tcp` provider is added (replaces `libfabric/sockets`)
+- UCX support is added (Technology Preview - not recommended in production environments)
```
Comment (johannlombardi) [inline docs/release/release_notes_v2_2.md:17]: s/recommended/supported ?

File: docs/release/release_notes_v2_2.md [hunk 5]
```diff
@@ -0,0 +1,134 @@
...
+
+CentOS Linux 8 support has been removed, and has been replaced by Rocky Linux 8 support.
+RHEL 8.5 is no longer supported by RedHat; RHEL 8.6 and RHEL 8.4 EUS are still supported.
+
+#### New Network Providers
+
+- Support for the `libfabric/tcp` provider is added.
+  This replaces the `libfabric/sockets` provider that was supported with DAOS 2.0.
+  The `tcp` provider achieves significantly higher performance than the `sockets` provider
+  and is the recommended provider for non-RDMA networks.
```
Comment (johannlombardi) [inline docs/release/release_notes_v2_2.md:34]: maybe we should claim that the socket provider isn't supported any longer then?

**Review by johannlombardi — APPROVED** [review]
Looks good to me. Just minor suggestion.

File: docs/release/release_notes_v2_2.md (hunk 4 above)
Comment (Michael-Hennecke) [reply docs/release/release_notes_v2_2.md:17]: done

File: docs/release/release_notes_v2_2.md (hunk 5 above)
Comment (Michael-Hennecke) [reply docs/release/release_notes_v2_2.md:34]: done

**Review by ipoddubn — APPROVED** [review]

**Review by johannlombardi — APPROVED** [review]

**Pull request merged**